doc/api/net.md

```diff
@@ -355,14 +443,23 @@ Users who experience large or growing `bufferSize` should attempt to
 "throttle" the data flows in their program with [`pause()`][] and [`resume()`][].
 
 ### socket.bytesRead
+<!-- YAML
+added: v0.1.98
+-->
```

Comment:
Are you sure? I think the first time this appears as a property in the source code is in 0.5.10